core/store/src/db/rocksdb.rs

```diff
@@ -23,6 +23,11 @@ pub(crate) mod snapshot;
 /// In the end, they are exported as Prometheus metrics.
 pub const CF_STAT_NAMES: [&'static str; 1] = [::rocksdb::properties::LIVE_SST_FILES_SIZE];
 
+/// How many writes before we execute a pre-write check.
+const CHECK_FREE_SPACE_INTERVAL: u16 = 256;
+/// How much free space is required for a write to be allowed.
+const FREE_SPACE_THRESHOLD: bytesize::ByteSize = bytesize::ByteSize::mb(16);
```

Comment:
tbh, this feels somewhat lower than it needs to be. I think some of our structs are larger than that? Chunks in particular.

Reply:
Oh, yes, definitely.  That’s also part of why I want to just get rid of the code in #7545 since I believe it’s broken.  I just didn’t want to change the values in this PR.